# Object relative position of camera
geometry_msgs/Pose pose

# Height of object [m]
float64 height

# Width of object [m]
float64 width

# Pick & place priority
int32 order
